main.c

```diff
@@ -22,7 +24,13 @@
 
 #define CLAMP2BYTE(v) (((unsigned) (v)) < 255 ? (v) : (v < 0) ? 0 : 255)
 
-unsigned int detect(uint8_t *pixel, int width, int height, int channels)
+#define OPT_PLANE 1
```

Comment:
This macro can be set via Makefile. That is, replace `-DENABLE_OPENMP` to `-DOPT_PLANE`. I will write scripts to detect OpenMP support.

Reply:
done